util/devel/chplspell-dictionary

```diff
@@ -5641,7 +5655,9 @@ acct
 
 FILEID: e72aeca4-45ab-11ec-a187-336475c959d4
 doit
+gpuized
 griddim
+uization
```

Comment:
"uization" does not sound right. ???

Reply:
That's from

```
./compiler/optimizations/gpuTransforms.cpp:
    // GPUization failed (it's an implementation detail). Instead, we want
```

I tend to err on the side of accepting what's written unless it's objectively wrong.  Another option would be to change that comment to `Gpuization` or `gpuization`, which are both used elsewhere in the Chapel source.

Reply:
I changed the comment to "gpuization" per offline discussion.